### ISSUE TITLE
Can't use this item

### ISSUE THREAD

**Issue by Jasonhohn** [Altevir/xamarinforms-gaming-store#2]
Sorry, after the project was opened, an unknown error occurred in the test.

Because of MAX_PATH, Java type cannot be generated for category AndroidX.Core.View.Accessibiity.AccessibilityManagerCompat/IAccessibilityStateChangeListenerImplementor: System.IO.DirectoryNotFoundException: Cannot find path'C:\Users\user\Downloads\xamarinforms-game-store-master\ Part of xamarinforms-game-store-master\XFGamingStore\XFGamingStore.Android\obj\Debug\100\android\src\mono\androidx\core\view\accessibility\AccessibilityManagerCompat_AccessibilityStateChangeListenerImplementor.java'.

Can you please tell me how to use it? Thank you
